scipy/stats/_continuous_distns.py

```diff
@@ -4195,6 +4195,101 @@ def _stats(self):
     def _entropy(self):
         return np.log(2)+1
 
+    def fit(self, data, *args, **kwds):
+        """
+        Maximum likelihood estimate for the location and scale parameters.
+
+        Exact forumulas are used to calculate the loc and scale, unless they
+        are provided in floc or fscale.
+
+        Parameters
+        ----------
+        data : array_like
+            Data to use in calculating the maximum likelihood estimate.
+        floc : float, optional
+            Hold the location parameter fixed to the specified value.
+        fscale : float, optional
+            Hold the scale parameter fixed to the specified value.
+
+        Returns
+        -------
+        loc, scale : float
+            Maximum likelihood estimates for the location and scale.
+
+        Notes
+        -----
+        An error is raised if floc and fscale are given. A warning is raised
+        if a provided parameter does not match MLE equations to indicate that
```

Comment:
```suggestion
        if either `floc` or `fscale` is provided but does not match the exact MLE value;
```
in this case `fit` falls back to the generic implementation.